app/controllers/idv/review_controller.rb

```diff
@@ -74,10 +77,15 @@ def create
       session[:last_gpo_confirmation_code] = idv_session.gpo_otp
     end
 
+    def step_indicator_step
+      return :secure_account unless address_verification_method == 'gpo'
+      :get_a_letter
+    end
+
     private
 
     def address_verification_method
-      user_session.dig('idv', 'address_verification_mechanism')
+      user_session.with_indifferent_access.dig('idv', 'address_verification_mechanism')
```

Comment:
Controller tests won't work because an idv_session is mapped as a symbol rather than a string. I'm not sure why it was specified as a string in the first place, but new tests of the `address_verification_mechanism` being set will not pass. I thought indifferent access would rock the boat the least